### ISSUE TITLE
Wakatime does not work on vscode.dev/github.dev

### ISSUE THREAD

**Issue by zumbiepig** [wakatime/vscode-wakatime#414]
When I install the Wakatime extension on VS Code for the Web (**not** GitHub Codespaces), I get an error saying `api key not provided` or `api key invalid`. I have properly configured my API key and API URL in the Settings and config.json.

**Comment by alanhamlett** [comment]
Type `cmd/ctrl + shift + p` then type `wakatime api key`. Does the value displayed match the one at https://wakatime.com/api-key?

It's working for me, and we don't have a `config.json` file related to vscode-wakatime on the web that I know of.

**Comment by zumbiepig** [comment]
> Type `cmd/ctrl + shift + p` then type `wakatime api key`. Does the value displayed match the one at https://wakatime.com/api-key?
> 
> It's working for me, and we don't have a `config.json` file related to vscode-wakatime on the web that I know of.

Yes, the value matches. The config.json that I am talking about is VS Code's config.json (Go to Settings > click the file icon in the top right). In VS Code settings, i search for wakatime and all the settings for wakatime (only 2) are correct.

**Comment by zumbiepig** [comment]
@alanhamlett I have heard from others that the `wakatime-cli` binary might be required for Wakatime to work properly. But that doesn't seem possible in a browser setting, and the VS Code Settings for the extension do explicitly mention `Defaults to ~/.wakatime.cfg, except in a browser.`

**Comment by alanhamlett** [comment]
The browser version uses a separate web-version of the plugin that we modified to not require wakatime-cli:

https://github.com/wakatime/vscode-wakatime/tree/master/src/web
